### PR TITLE
Pin `pandoc` to suggested versions.

### LINKED ISSUE #22: pandoc dependency should be version range pinned
with a fresh nbconvert install, one will drag in the latest version of pandoc, which seems unsupported:

```
Your version must be at least (1.12.1) but less than (2.0.0).
Refer to http://pandoc.org/installing.html.
Continuing with doubts...
  check_pandoc_version()
```

### PR DESCRIPTION
**Pull request by ocefpaf**
Should fix #22

### CODE REVIEW

**Comment by ocefpaf** [comment]
Thanks @marscher for reporting this.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ requirements:
     - pandocfilters >=1.4.1
     - testpath
     - defusedxml
-    - pandoc
+    - pandoc >=1.12.1,<2.0.0
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @msarahan, @jjhelmus, and @mingwandroid this is something you may want to fix in the AR version of the recipe too.

**Comment by marscher** [comment]
thanks for fixing it so quickly :+1:

**Comment by jakirkham** [comment]
Thanks for doing this. Unfortunately am seeing issues with the solver wiggling out of this constraint by downgrading `nbconvert` to allow a newer version of `pandoc`. Am hesitant to mark old `nbconvert` packages as `broken`, but don't know what else to do. Thoughts?

**Comment by jjhelmus** [comment]
The requirements of the older `converter` packages could be patched by adding a rule to the [conda-forge-repodata-patches-feedstock](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock).  I'm not certain if repodata patching is live yet on Anaconda.org but this would be a great test case.

**Pull request merged**